admin/mkassoc

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env ruby
 
 require "openid/consumer/associationmanager"
-require "openid/store/memstore"
+require "openid/store/memory"
 
-store = OpenID::MemoryStore.new
+store = OpenID::Store::Memory.new
 ARGV.each do |server_url|
-  mgr = OpenID::Consumer::AssociationManager.new(store, URI.parse(server_url))
+  mgr = OpenID::Consumer::AssociationManager.new(store, server_url)
```

Comment:
@tobiashm 
Thank you for your comment!

If `OpenID::Consumer::AssociationManager#get_association` with `@server_url` of URI object is called, `bad URI` error occured like the following:

```sh
$ bundle exec ruby admin/mkassoc http://auth.livedoor.com/openid/server
==================================================
Server: http://auth.livedoor.com/openid/server
/Users/seri/Documents/ruby-work/ruby-openid/lib/openid/kvpost.rb:54:in `rescue in make_kv_post': Unable to contact OpenID server: bad URI(is not URI?): http://
auth.livedoor.com/openid/server (OpenID::KVPostNetworkError)
        from /Users/seri/Documents/ruby-work/ruby-openid/lib/openid/kvpost.rb:51:in `make_kv_post'
        from /Users/seri/Documents/ruby-work/ruby-openid/lib/openid/consumer/associationmanager.rb:195:in `request_association'
        from /Users/seri/Documents/ruby-work/ruby-openid/lib/openid/consumer/associationmanager.rb:130:in `negotiate_association'
        from /Users/seri/Documents/ruby-work/ruby-openid/lib/openid/consumer/associationmanager.rb:118:in `get_association'
        from admin/mkassoc:11:in `block in <main>'
        from admin/mkassoc:7:in `each'
        from admin/mkassoc:7:in `<main>
```

This is because `URI.parse` is called twice, 1st time is [here](https://github.com/openid/ruby-openid/blob/master/admin/mkassoc#L8), and second time is [here](https://github.com/openid/ruby-openid/blob/master/lib/openid/fetchers.rb#L183).

So, if you want to validate `server_url` in `admin/mkassoc`, how about adding this?

```ruby
ARGV.each do |server_url|
  unless URI::regexp =~ server_url
    puts "`#{server_url}` will be skipped for invalid URI format."
    next
  end
  ...

end
```